ext/standard/tests/hrtime/hrtime.phpt

```diff
@@ -0,0 +1,20 @@
+--TEST--
+Test hrtime() returns nanoseconds
+--FILE--
+<?php
+
+$time = hrtime();
+usleep(500000);
```

Comment:
Testing with any kind of sleep is not reliable, sleep might freeze the process on some system. You might be much better served checking some block execution dynamically, and comparing with another timing implementation like microtime(). Here fe what i invented for the hrtime ext http://svn.php.net/viewvc/pecl/hrtime/trunk/tests/helper.inc?view=markup . In general, it also sohuld be compared to an existing time implementation in PHP, to ensure the compatibility and correctness.

Thanks.